### PR TITLE
feat: openid4vp flutter integration.

### PR DESCRIPTION
**Pull request by vkubiv**
Signed-off-by: Volodymyr Kubiv <volodymyr.kubiv@euristiq.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-sdk-gomobile/didresolver/resolver.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ SPDX-License-Identifier: Apache-2.0
 package didresolver
 
 import (
+	// helps gomobile bind api.DIDResolver interface to Resolver implementation in ios-bindings.
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/didresolver/resolver.go:11]: Just curious what issue you ran into here? (I don't see any problem with doing this, just want to know for my own knowledge)

File: demo/app/android/app/src/main/kotlin/walletsdk/openid4vp/OpenID4VP.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    fun presentCredential(signingKeyId: String) {
+        val verifiablePresentation = this.verifiablePresentation
+                ?: throw Exception("OpenID4VP interaction not properly initialized, call processAuthorizationRequest first")
+        val initiatedInteraction = this.initiatedInteraction
+                ?: throw Exception("OpenID4VP interaction not properly initialized, call processAuthorizationRequest first")
+
+        initiatedInteraction.presentCredential(verifiablePresentation, signingKeyId)
+    }
+}
```
Comment (DRK3) [inline demo/app/android/app/src/main/kotlin/walletsdk/openid4vp/OpenID4VP.kt:51]: Ideally should have newline at end of file

**Review by DRK3 — APPROVED** [review]
Minor comments about missing newlines

File: cmd/wallet-sdk-gomobile/didresolver/resolver.go (hunk 1 above)
Comment (vkubiv) [reply cmd/wallet-sdk-gomobile/didresolver/resolver.go:11]: without this import, it generates Resolve that not implements the DIDResolve interface. 
Previously it was :
`@interface DidresolverResolver : NSObject <goSeqRefInterface> {
}`

With import it becomes

`@interface DidresolverResolver : NSObject <goSeqRefInterface, ApiDIDResolver> {
}` 

so now DidresolverResolver can be cast to ApiDIDResolver

Comment (vkubiv) [reply cmd/wallet-sdk-gomobile/didresolver/resolver.go:11]: Just to easily understand this is wired Objective-C syntax, analog on Java will look like that.

`class DidresolverResolver implements ApiDIDResolver> {  }`

**Pull request merged**